.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,14 @@
-This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.
+Thanks for your interest in The Odin Project. In order to get PRs closed in a reasonable amount of time, we request that you include a baseline of information about hthe changes you are proposing. Please answer the following triage quesions:
```

Comment:
I see a lot of open source projects providing check lists in their PR templates. Do you think those would be more user friendly for creating PR's?

Maybe something like this at the start:

### Pre Check List
- [ ] [You have read our contributing guide]()
- [ ] This Pull request resolves an existing open issue - [if not please discuss what you want to do with the maintainers first](https://discord.com/channels/505093832157691914/540903304046182425)
etc

I want to add some tooling to the site that will allow contributors to "preview" thier content with our site markdown parser applied. That could easily be added to a checklist like this.

Reply:
I've also noticed we don't have a contributing guide on this repo 🙈 . We did have one at one point. In any case we need something similar to what we have on the web app: https://github.com/TheOdinProject/theodinproject/wiki/Contributing-Guide

Reply:
@KevinMulhern See #21812 re: contributing guide

There are some other niceties we can add according to the [official guide](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community) but we can move those discussions to separate issues or the discussion repo.